resources/templates/account.html

```diff
@@ -53,31 +53,19 @@
     <div class="tooltip">
       <ul>
         <li>
-          Raiden accounts are Ethereum accounts defined by a pair of keys, a
-          private key and public key.
+          Download you keystore file now. You can restore your Raiden account
+          with this file and the password via Metamask.
         </li>
         <li>
-          Every private key/address pair is encoded in a keyfile. Keyfiles are
-          JSON text files which you can open and view in any text editor.
+          Send 0.125 ETH to your Raiden account. 0.075 ETH are used as safety buffer
+          to operate 3 channels. 0.05 ETH are needed for RDN and DAI tokens.
```

Comment:
Here the calculation is confusing. 0.075 + 0.05 = 0.08.  Maybe it makes sense to add also information for what the rest is needed (transaction fees and buffer)

Reply:
0.075 + 0.050 (=0.05) = 0.125, oder? Ich glaube transaction fees and buffer verwirrt die Leute

Reply:
oh verrechnet
alles gut